packages/dds/sequence/src/test/intervalCollection.fuzzUtils.ts

```diff
@@ -138,6 +152,13 @@ export function makeReducer(loggingInfo?: LoggingInfo): Reducer<Operation, Clien
 			const collection = channel.getIntervalCollection(collectionName);
 			collection.changeProperties(id, { ...properties });
 		},
+		revertSharedStringRevertibles: async ({ channel }) => {
```

Comment:
reducer logic should be deterministic. All random choices should happen at operation generation time. So for this example, the `RevertOperation` should say how many edits should be reverted, and that should just be read here.

This has the advantage of failure JSON clearly specifying exactly what happened as part of the fuzz test.
Using the randomness source inside the fuzz test rather than `Math.random()` ensures that the tests are deterministic, which is critical for reproducibility.